specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-09-02-preview/examples/Usages_List.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "api-version": "2023-09-02-preview",
+    "subscriptionId": "subid1",
+    "location": "location1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "currentValue": 8,
+            "limit": 1000,
+            "name": {
+              "localizedValue": "Numbers of ManagedClusters",
```

Comment:
Is this correct? "LocalizedValue" makes me think of string localization/internationalization, rather than "friendly name" and "not friendly name"

Reply:
Maybe `Count of Managed Clusters`?

Reply:
1. there is a unit in the payload.
2. I am thinking of just have localizedvalue:"Managed Clusters", and value: "ManagedClusters"